src/pages/bukime-budrus/index.js

```diff
@@ -30,7 +30,7 @@ const Page = ({ data }) => {
 
       {!!content && (
         <Constraint>
-          <Breadcrumb className="__solo" crumbs={crumbs} />
```

Comment:
The solo thing is not even needed. Wrap breadcrumbs in `NavigationGroup` to apply the same styling as in other pages.